### DISCUSSION TITLE
A tldr-pages client written in Bash

### DISCUSSION THREAD

**Discussion by ghost** [tldr-pages/tldr#7624] [Show and tell]
Hello everyone! I've made a tldr-pages client written in Bash. It's still a work in progress and any help in developing it will be highly appreciated.

tldr-bash is deprecated. Please use the official tldr-pages client instead.

**Comment by CleanMachine1** [comment]
Hey, where did it go?
The link is 404

**Reply by ghost** [reply]
There is another tldr-pages client written in Bash that have been completed and works fine, and tldr-bash was incomplete and very limited in terms of features.

**Reply by git-em** [reply]
Could you post the link to the other client?

**Reply by ghost** [reply]
@git-em [raylee/tldr-sh-client](https://github.com/raylee/tldr-sh-client). It's written in POSIX Shell so it will be more compatible with other shells.

**Comment by CleanMachine1** [comment]
Since the client has been discontinued, I am locking this discussion.

**Reply by sbrl** [reply]
Source for the client being discontinued?

**Reply by CleanMachine1** [reply]
Authors edited original comment.
